packages/common/src/api/user.ts

```diff
@@ -68,7 +68,6 @@ const userApi = createApi({
         return apiUser?.[0]
       },
       options: {
-        idArgKey: 'handle',
```

Comment:
Sup with this?

Reply:
That isn't the right value to be passing. `idArgKey` should be the name of the arg containing the id of the entity you're getting.

As written, it would be looking for the handle as an existing id in the users cache. We don't have a way to do that for non-id fields currently, so removing it is correct. Just a campground improvement.